resources/assets/coffee/forum/topic-ajax.coffee

```diff
@@ -19,23 +19,44 @@ $(document).on 'ajax:success', '.delete-post-link', (_event, data) ->
   $el = $(".js-forum-post[data-post-id=#{data.postId}]")
   currentHeight = $el.css 'height'
 
-  $el
-    .css
-      minHeight: '0px'
-      height: currentHeight
-    .slideUp null, ->
-      $el.remove()
-
-      window.forum.setTotalPosts(window.forum.totalPosts() - 1)
-
-      for post in window.forum.posts by -1
-        originalPosition = parseInt post.getAttribute('data-post-position'), 10
-
-        break if originalPosition < data.postPosition
-
-        post.setAttribute 'data-post-position', originalPosition - 1
-
-      osu.pageChange()
+  if currentUser.isAdmin || currentUser.isGMT
+    $post = $el.find '.forum-post'
+    toggle = _event.target
+
+    switch toggle.id
+      when 'delete'
+        $post.addClass 'forum-post--hidden'
```

Comment:
i probably shouldn't be renaming a whole [bem class](https://github.com/ppy/osu-web/blob/master/resources/assets/less/bem/forum-post.less)

also this isn't really js-related, just applied in js